### PR TITLE
Update tiered cache documentation

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Update tiered cache documentation

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _search-plugins/caching/tiered-cache.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ nav_order: 10
...
+A tiered cache is a multi-level cache in which each tier has its own characteristics and performance levels. By combining different tiers, you can achieve a balance between cache performance and size.
 
 ## Types of tiered caches
 
-OpenSearch 2.13 provides an implementation of _tiered spillover cache_. This implementation spills the evicted items from upper to lower tiers. The upper tier is smaller in size but offers better latency, like the on-heap tier. The lower tier is larger in size but is slower in terms of latency compared to the upper tier. A disk cache is an example of a lower tier. OpenSearch 2.13 offers on-heap and disk tiers. 
+OpenSearch 2.13 provides an implementation of _tiered spillover cache_. This implementation spills the evicted items from upper to lower tiers. The upper tier, such as the on-heap tier, is smaller in size but offers better latency. The lower tier, such as a disk cache, is larger in size but is slower in terms of latency. OpenSearch 2.13 offers on-heap and disk tiers.
```
Comment (natebower) [inline _search-plugins/caching/tiered-cache.md:18]:
```suggestion
OpenSearch 2.13 provides an implementation of a _tiered spillover cache_. This implementation spills the evicted items from upper to lower tiers. An upper tier, such as an on-heap tier, is smaller in size but offers better latency. A lower tier, such as a disk cache, is larger in size but is slower in terms of latency. OpenSearch 2.13 offers on-heap and disk tiers. 
```

File: _search-plugins/caching/tiered-cache.md [hunk 2]
```diff
@@ -41,42 +41,44 @@ In OpenSearch 2.13, a request cache can use a tiered cache. To begin, configure
...
+Set the on-heap and disk store tiers as follows:
 
 ```yaml
 indices.request.cache.tiered_spillover.onheap.store.name: opensearch_onheap
 indices.request.cache.tiered_spillover.disk.store.name: ehcache_disk
 ```
 {% include copy.html %}
 
-For more information about installing non-bundled plugins, see [Additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/#additional-plugins).
+The `opensearch_onheap` setting is the built-in on-heap cache available in OpenSearch.
```
Comment (natebower) [inline _search-plugins/caching/tiered-cache.md:61]: The setting can't literally be the cache.

File: _search-plugins/caching/tiered-cache.md [hunk 3]
```diff
@@ -41,42 +41,44 @@ In OpenSearch 2.13, a request cache can use a tiered cache. To begin, configure
...
 ```yaml
 indices.request.cache.tiered_spillover.onheap.store.name: opensearch_onheap
 indices.request.cache.tiered_spillover.disk.store.name: ehcache_disk
 ```
 {% include copy.html %}
 
-For more information about installing non-bundled plugins, see [Additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/#additional-plugins).
+The `opensearch_onheap` setting is the built-in on-heap cache available in OpenSearch. 
+
+The `ehcache_disk` setting is the disk cache implementation from [Ehcache](https://www.ehcache.org/). Using `ehcache_disk` requires installing the `cache-ehcache` plugin.
```
Comment (natebower) [inline _search-plugins/caching/tiered-cache.md:63]: The setting can't literally be the disk cache implementation.

**Review by natebower — COMMENTED** [review]
@kolchfa-aws

File: _search-plugins/caching/tiered-cache.md (hunk 2 above)
Comment (kolchfa-aws) [reply _search-plugins/caching/tiered-cache.md]: Reworded.

File: _search-plugins/caching/tiered-cache.md (hunk 3 above)
Comment (kolchfa-aws) [reply _search-plugins/caching/tiered-cache.md]: Reworded.

**Review by natebower — APPROVED** [review]
LGTM

**Pull request merged**